### ISSUE TITLE
Add o, o2, n2 and temperature profile in chgres_cube.fd for WAM IC

### ISSUE THREAD

**Issue by HenryJuang-NOAA** [NOAA-EMC/UFS_UTILS#164]
We would like to create code to generates o, o2, n2 and temperatire profiles based on input date for upper atmosphere from mesopause up to thermosphere tor FV3 WAM IC.

**Comment by HenryJuang-NOAA** [comment]
We have implemented the capability in chgres_cube for adding statistical values of o, o2 distribution and temperature (T) over thermosphere to giving cold start initial condition (IC) for WAM (Whole Atmosphere Model) with vertical layer up to thermosphere with layer number 150. To provide statistical fields of o, o2 and temperature field over thermosphere, a group of subroutines in a file, and a subroutine called VINTG_WAM was coded in atmosphere.F90 to compute all new fields of o, o2, and T based on IC date over thermosphere. In addition to provide new tracers o and o2 in configure nml file in issue #163, two parameters were given to control whether WAM IC will need or not in running chgres_cube, and when is the IC for generating required new fields based on IC date. These two parameters are wam_cold_start with true or false options, and wam_start_date with yyyymmddhh.

**Comment by edwardhartnett** [comment]
Do we have a way to test?

**Comment by HenryJuang-NOAA** [comment]
This issue can be closed. The main issue for this has been resolved by using the package called NRLMSISE00 to extend atmospheric model top up to thermosphere as WAM IC. The package is a neutral atmosphere empirical model to provide standard atmospheric profile from surface to exosphere for atmospheric main gases such as O, O2, O3, and N2 and temperature fields, as function of time, latitude and longitudes. The package is originally written in FORTRAN 77 with BLOCKDATA and common statement. Due to GNU compiler, we have changed BLOCKDATA and COMMON statement into module and use module statement. It has been renamed as wam_climo_data as filename for easy to locate, and the option to have WAM IC with this data is by passing wam_cold_start=.T. to run chgres_cube.  All these have been tested success as described in issue #165 for FV3 WAM. This issue should be closed.
